src/parser/flow_lexer.ml

```diff
@@ -58,6 +58,14 @@ let scinumber = [%sedlex.regexp?
 let wholenumber = [%sedlex.regexp? underscored_digit, Opt '.']
 let floatnumber = [%sedlex.regexp? Opt underscored_digit, '.', underscored_decimal]
 
+let binbigint = [%sedlex.regexp? binnumber, 'n']
+let octbigint = [%sedlex.regexp? octnumber, 'n']
+let legacyoctbigint = [%sedlex.regexp? legacyoctnumber, 'n']
+let hexbigint = [%sedlex.regexp? hexnumber, 'n']
+let scibigint = [%sedlex.regexp? decintlit, ('e' | 'E'), Opt ('-' | '+'), underscored_digit, 'n']
```

Comment:
Why not `[%sedlex.regexp? scinumber, 'n']`? Since `scibigint` isn't supported by the proposal, why does this differ from `scinumber`?